rest-service/manager_rest/resource_manager.py

```diff
@@ -1381,6 +1383,105 @@ def _finalize_create_deployment(self, deployment: models.Deployment):
         # RD-1603 will move this:
         self.create_deployment_schedules(deployment, deployment.blueprint.plan)
 
+    @staticmethod
+    def get_deployment_parents_from_labels(labels):
+        parents = []
+        for label, value in labels:
+            if label == 'csys-obj-parent':
+                parents.append(value)
+        return parents
+
+    def verify_deployment_parent_labels(self, parents, deployment_id):
+        if not parents:
+            return
+        missing_parents = []
+        result = self.sm.list(
+            models.Deployment,
+            include=['id'],
+            filters={'id': lambda col: col.in_(parents)}
+        ).items
+        _existing_parents = [_parent[0] for _parent in result]
+        for parent in parents:
+            if parent not in _existing_parents:
+                missing_parents.append(parent)
+
+        if missing_parents:
+            raise manager_exceptions.DeploymentParentNotFound(
+                'Deployment {0}: is referencing deployments'
+                ' using label `csys-obj-parent` that are not existed, '
+                'make sure that deployments {1} are existed before create '
+                'deployment'.format(deployment_id, ','.join(missing_parents))
+            )
+
+    def _place_deployment_label_dependency(self, source, target):
+        now = utils.get_formatted_timestamp()
```

Comment:
I don't think you need to provide this- it already defaults to the current time, so probably best just to not provide it except when it's being overridden (which this isn't).

Reply:
@geokala The reason why I'm adding this because the time is not inserted by default unfortunately if you check other models you will notice that and if I omitted this the `created_at` is not null by default https://github.com/cloudify-cosmo/cloudify-manager/blob/master/rest-service/manager_rest/storage/resource_models.py#L73 and I do not think so it set by default

Reply:
if you follow your link, you'll see that it already has a default :P

Reply:
Ohh thats so impressing, @tehasdf this is because I'm using pycharm and it redirect me to old env that has a code with default value